pkg/proxy/proxy_test.go

```diff
@@ -202,6 +202,42 @@ func TestProxy_List(t *testing.T) {
 			expectedContentType: "application/json; charset=utf-8",
 			expectedResponse:    "{\"error\":\"missing tag label\"}\n",
 		},
+		{
+			scenario:            "invalid phase",
+			urlParams:           "phase=foo",
+			result:              &apisLoadTestV1.LoadTestList{},
+			expectedCode:        200,
+			expectedContentType: "application/json; charset=utf-8",
+			expectedResponse:    "{\"limit\":0,\"continue\":\"\",\"remain\":null,\"items\":[]}\n",
```

Comment:
you can use back ticks "`" to avoid manually escaping all internal double-quotes, it will make cases clearer
```suggestion
			expectedResponse:    `{"limit":0,"continue":"","remain":null,"items":[]}\n`,
```